<!DOCTYPE html>
<html>
<head>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Bohdan Khomtchouk</title>
<link href="bootstrap/css/bootstrap.min.css" rel="stylesheet">
<link rel="stylesheet" href="css/styles.css"/>
<link href="bootstrap/css/bootstrap-responsive.min.css" rel="stylesheet">
<link href="FontAwesome/css/font-awesome.min.css" rel="stylesheet">
<link rel="stylesheet" href="css/academicons.css"/>
</head>
<body>

<!-- nav bar -->
<div class="navbar navbar-inverse navbar-fixed-top">
	<div class="navbar-inner">
		<div class="container">
			<button type="button" class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
			</button>
			<a class="brand" href="#">Bohdan Khomtchouk</a>
			<div class="nav-collapse collapse">
				<ul class="nav">
					<li class="active"><a href="#">Home</a></li>
					<li><a href="#about">About</a></li>
					<li><a href="#publications">Publications</a></li>
					<li><a href="#packages">Packages</a></li>
					<li><a href="#projects">Projects</a></li>
					<li><a href="#conferences">Conferences</a></li>
					<li><a href="#awards">Awards</a></li>
					<li><a href="#alumni">Alumni</a></li>
					<li><a href="#press">Press</a></li>
					<li><a href="#contact">Contact</a></li>
				</ul>
			</div><!--/.nav-collapse -->
		</div>
	</div>
</div>

<div class="container" id="main">
	<div class="row">
	
		<!-- right part of page -->
		<div class="hidden-phone">

			<div class="span4">
				<img style="float: right;" src="img/bohdan.JPG">
			</div>
		</div>

		<!-- left part of page -->
		<div class="span8">
			<h1 style="color: #445555;">Bohdan Khomtchouk</h1>
			<p class="lead">Bioinformatician at the University of Miami Miller School of Medicine</p>
			<blockquote>
				<p style="font-size: 11pt">
					Triple-major summa cum laude (B.Sc. Mathematics, B.Sc. Physics, B.Sc. Molecular Biology and Biochemistry), currently US Department of Defense NDSEG Fellow pursuing Ph.D. in computational human genetics and genomics.  
					Textbook author (CRC Press) in organic chemistry, applied mathematician, biostatistician, and bioinformatics software engineer.
				</p>
				<!--
					<small>Josh Wills of <a href="http://www.cloudera.com/content/cloudera/en/home.html">Cloudera</a></small>
				-->
			</blockquote>
			<div id="about">
				<h2>About Me</h2>
				<p>
					I am a doctoral candidate funded by the <a href="https://ndseg.asee.org">National Defense Science & Engineering Graduate (NDSEG) Fellowship</a>, working in the laboratory
					of <a href="https://scholar.google.com/citations?user=sPhaqa0AAAAJ&hl=en">Prof. Dr. Claes Wahlestedt M.D., Ph.D.</a> at the University of Miami Miller School of Medicine in Miami, FL USA.
				    I am graduating in Summer 2017 with a Ph.D. in <a href="http://biomed.med.miami.edu/graduate-programs/human-genetics-and-genomics">Human Genetics and Genomics (computational track)</a>.  My dissertation title is:
					"Developing mathematical and computational epigenetic software packages and multidimensional analysis pipelines for ChIP-seq and RNA-seq studies in the genomic fight against alcoholism."
					My published thesis works to-date include a <a href="http://www.nature.com/mp/journal/vaop/ncurrent/full/mp2016131a.html">Molecular Psychiatry paper</a>, 
					<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1260-x">BMC Bioinformatics paper</a>, <a href="http://bib.oxfordjournals.org/content/early/2016/12/31/bib.bbw130.short?rss=1">Briefings in Bioinformatics paper</a>, <a href="http://scfbm.biomedcentral.com/articles/10.1186/s13029-014-0030-2">Source Code for Biology and Medicine paper</a>, <a href="http://jaha.ahajournals.org/content/5/12/e004076">Journal of the American Heart Association paper</a>, <a href="https://bioconductor.org/packages/geneXtendeR/">R/Bioconductor package</a>, and <a href="https://pypi.python.org/pypi/FTPwalker/0.1">Python/PyPI package</a>.
					I have also published an <a href="https://www.crcpress.com/Survival-Guide-to-Organic-Chemistry-Bridging-the-Gap-from-General-Chemistry/McMahon-Khomtchouk-Wahlestedt/p/book/9781498777070">organic chemistry textbook</a> at CRC Press (Taylor & Francis).
					In addition, I have released several preprints, which are currently under peer-review.  These preprints span the fields of mathematical genetics and evolutionary biology (<a href="http://dx.doi.org/10.1101/076679">here</a>, <a href="http://dx.doi.org/10.1101/066381">here</a>, 
					and <a href="http://arxiv.org/abs/1402.4095">here</a>), databases and search engines (<a href="http://dx.doi.org/10.1101/069575">here</a>), HPC graphics visualization (<a href="http://dx.doi.org/10.1101/076463">here</a>), 
					and nonlinear physics (<a href="https://arxiv.org/abs/1603.03153">here</a>).
				</p>
					
				<p>
					My thesis committee chair is <a href="http://ccs.miami.edu/team_member/nicholas-tsinoremas-phd/">Prof. Dr. Nicholas Tsinoremas Ph.D.</a>  My thesis committee members are: 
					<a href="http://ccs.miami.edu/?p=1259">Prof. Dr. Mitsunori Ogihara Ph.D.</a>, <a href="http://uhealthsystem.com/researchers/profile/95468">Prof. Dr. Eden Martin Ph.D.</a>, and 
					<a href="http://centeronaging.med.miami.edu/pharvey">Prof. Dr. Phil Harvey Ph.D.</a>.
					My biophysics advisor is <a href="https://scholar.google.com/citations?user=2Tl77HMAAAAJ&hl=en">Prof. Dr. Wolfgang Nonner M.D.</a> and 
					my bioinformatics advisor is <a href="http://www.pubfacts.com/author/Derek+Van+Booven">Derek Van Booven</a>, Senior Bioinformatics Analyst at the <a href="http://hihg.med.miami.edu">John P. Hussman Institute for Human Genomics</a>.  
					My academic appointments are in the <a href="http://psychiatry.med.miami.edu/research/center-for-therapeutic-innovation">Center for Therapeutic Innovation</a> and 
					the <a href="http://psychiatry.med.miami.edu">Department of Psychiatry and Behavioral Sciences</a>.
				</p>
				
				<p>
				    I am married to the best woman in the world, Dr. Khomtchouk Ph.D. (Microbiology and Immunology).  We have one phenomenal son, Sviatoslav, to whom I have dedicated <a href="https://www.crcpress.com/Survival-Guide-to-Organic-Chemistry-Bridging-the-Gap-from-General-Chemistry/McMahon-Khomtchouk-Wahlestedt/p/book/9781498777070">my organic chemistry textbook</a>.
				</p>
			</div>
			
			<div id="publications">
				<h2>Publications</h2>
				<h3>Books</h3>
				<p>
				<strong>
					Survival Guide to Organic Chemistry: Bridging the Gap from General Chemistry
				</strong>
					(<a href="https://www.crcpress.com/Survival-Guide-to-Organic-Chemistry-Bridging-the-Gap-from-General-Chemistry/McMahon-Khomtchouk-Wahlestedt/p/book/9781498777070">CRC Press (Taylor & Francis), 2016, pp. 1-684</a>)<br />
				<small>
					Patrick E. McMahon, Bohdan B. Khomtchouk, Claes Wahlestedt
				</small>
				</p>
				
				<h3>Articles</h3>
				<p>
				<strong>
					MicroScope: ChIP-seq and RNA-seq software analysis suite for gene expression heatmaps
				</strong>
					(<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1260-x">BMC Bioinformatics, 2016, 17:390, pp. 1-9</a>)<br />
				<small>
					Bohdan B. Khomtchouk, James R. Hennessy, Claes Wahlestedt
				</small>
				</p>
				<strong>
					Dependence-induced increase of alcohol self-administration and compulsive drinking mediated by the histone methyltransferase PRDM2
				</strong>
					(<a href="http://www.nature.com/mp/journal/vaop/ncurrent/full/mp2016131a.html">Molecular Psychiatry, 2016, doi: 10.1038/mp.2016.131, pp. 1-13</a>)<br />
				<small>
					Estelle Barbier, Andrea Johnstone, Bohdan Khomtchouk, Jenica Tapocik, Caleb Pitcairn, Faazal Rehman, Eric Augier, Abbey Borich, Jesse Schank, Christopher Rienas, Derek Van Booven, Sun Hui, Daniel Natt, Claes Wahlestedt, Markus Heilig
				</small>
				</p>
				<strong>
					Ischemic preconditioning confers epigenetic repression of Mtor and induction of autophagy through G9a-dependent H3K9 di-methylation
				</strong>
					(<a href="https://doi.org/10.1161/JAHA.116.004076">Journal of the American Heart Association, 2016, 5:12, pp. 1-27</a>)<br />
				<small>
					Olof Gidl&ouml;f, Andrea Johnstone, Kerstin Bader, Bohdan Khomtchouk, Jiaqi O'Reilly, Derek Van Booven, Claes Wahlestedt, Bernhard Metzler, David Erlinge
				</small>
				</p>
				<strong>
					How the Strengths of Lisp-Family Languages Facilitate Building Complex and Flexible Bioinformatics Applications
				</strong>
				(<a href="http://bib.oxfordjournals.org/content/early/2016/12/31/bib.bbw130.short?rss=1">Briefings in Bioinformatics, 2016, doi: 10.1093/bib/bbw130, pp. 1-7</a>)<br />
				<small>
					Bohdan B. Khomtchouk, Edmund Weitz, Peter D. Karp, Claes Wahlestedt
				</small>
				</p>
				<strong>
					HeatmapGenerator: high performance RNAseq and microarray visualization software suite to examine differential gene expression levels using an R and C++ hybrid computational pipeline
				</strong>
					(<a href="http://scfbm.biomedcentral.com/articles/10.1186/s13029-014-0030-2">Source Code for Biology and Medicine, 2014, 9:30, pp. 1-6</a>)<br />
				<small>
					Bohdan Khomtchouk, Derek Van Booven, Claes Wahlestedt
				</small>
				</p>
				
				<h3>Preprints</h3>
				<p>
				<strong>
					A global perspective of codon usage
				</strong>
					(<a href="http://dx.doi.org/10.1101/076679">bioRxiv (Cold Spring Harbor Laboratory), 2016, pp. 1-9</a>)<br />
				<small>
					Bohdan B. Khomtchouk, Claes Wahlestedt, Wolfgang Nonner
				</small>
				</p>
				<strong>
					Codon usage is a stochastic process across genetic codes of the kingdoms of life
				</strong>
					(<a href="http://dx.doi.org/10.1101/066381">bioRxiv (Cold Spring Harbor Laboratory), 2016, pp. 1-18</a>)<br />
				<small>
					Bohdan B. Khomtchouk, Claes Wahlestedt, Wolfgang Nonner
				</small>
				</p>
				<strong>
					PubData: search engine for bioinformatics databases worldwide
				</strong>
					(<a href="http://dx.doi.org/10.1101/069575">bioRxiv (Cold Spring Harbor Laboratory), 2016, pp. 1-8</a>)<br />
				<small>
					Bohdan B. Khomtchouk, Kasra A. Vand, Thor Wahlestedt, Kelly Khomtchouk, Mohammed K. Sayed, Claes Wahlestedt
				</small>
				</p>
				<strong>
					geneXtendeR: R/Bioconductor package for functional annotation of histone modification ChIP-seq data in a 3D genome world
				</strong>
					(<a href="http://dx.doi.org/10.1101/082347">bioRxiv (Cold Spring Harbor Laboratory), 2016, pp. 1-15</a>)<br />
				<small>
					Bohdan B. Khomtchouk, Derek Van Booven, Claes Wahlestedt
				</small>
				</p>
				<strong>
					shinyheatmap: ultra fast low memory heatmap software for big data genomics
				</strong>
					(<a href="http://dx.doi.org/10.1101/076463">bioRxiv (Cold Spring Harbor Laboratory), 2016, pp. 1-6</a>)<br />
				<small>
					Bohdan B. Khomtchouk, Claes Wahlestedt
				</small>
				</p>
				<strong>
					Zipf's law emerges asymptotically during phase transitions in communicative systems
				</strong>
					(<a href="http://arxiv.org/abs/1603.03153">arXiv (Cornell University Library), 2016, pp. 1-5</a>)<br />
				<small>
					Bohdan B. Khomtchouk, Claes Wahlestedt
				</small>
				</p>
				<strong>
					The mathematics of the genetic code reveal that frequency degeneracy leads to exponential scaling in the DNA codon distribution of Homo sapiens
				</strong>
					(<a href="http://arxiv.org/abs/1402.4095">arXiv (Cornell University Library), 2014, pp. 1-8</a>)<br />
				<small>
					Bohdan B. Khomtchouk
				</small>
			
			
			
			
			
				<div id="packages">
				
				<h2>Software packages</h2>
				
				<h4>
				geneXtendeR
					<small>	
						<a href="https://bioconductor.org/packages/geneXtendeR/">(Bioconductor)</a>
					</small>
					<small>
						<a href="https://github.com/Bohdan-Khomtchouk/geneXtendeR">(GitHub)</a>
					</small>
				</h4>
				<p>
					geneXtendeR is an R/Bioconductor package for histone modification ChIP-seq analysis. It is designed to optimally annotate a histone 
					modification ChIP-seq peak input file with functionally important genomic features (e.g., genes associated with peaks) 
					based on optimization calculations. geneXtendeR optimally extends the boundaries of every gene in a genome by some genomic 
					distance (in DNA base pairs) for the purpose of flexibly incorporating cis-regulatory elements, such as enhancers and 
					promoters, as well as downstream elements that are important to the function of the gene (relative to an epigenetic histone 
					modification ChIP-seq dataset). geneXtendeR computes optimal gene extensions tailored to the broadness of the specific epigenetic 
					mark (e.g., H3K9me1, H3K27me3), as determined by a user-supplied ChIP-seq peak input file. As such, geneXtendeR maximizes the 
					signal-to-noise ratio of locating genes closest to and directly under peaks. By performing a computational expansion of this nature, 
					ChIP-seq reads that would initially not map strictly to a specific gene can now be optimally mapped to the regulatory regions of the gene, 
					thereby implicating the gene as a potential candidate, and thereby making the ChIP-seq experiment more successful. Such an approach becomes 
					particularly important when working with epigenetic histone modifications that have inherently broad peaks.
				</p>
			</div>
				
				<h4>
				FTPwalker
					<small>
					<a href="https://badge.fury.io/py/FTPwalker"><img src="https://badge.fury.io/py/FTPwalker.svg" alt="PyPI version" height="18"></a>
					</small>
				</h4>
				<p>
					FTPwalker is a network programming Python package for optimally traversing extremely large FTP directory trees. 
					It constitutes the algorithmic heart of the <a href="https://github.com/Bohdan-Khomtchouk/PubData">PubData search engine</a>.
					FTPwalker creates a dictionary formatted as a JSON file in the user’s home directory containing all the full paths as keys 
					and the respective filenames as values.  FTPwalker is designed with speed in mind by utilizing state-of-the-art high performance 
					parallelism and concurrency algorithms to traverse huge directory trees through a remote network via an FTP connection.  The resultant hash table (i.e., dictionary) supports fast
					lookup for any file in any biological database (or any remote database with an extremely large file system).
				</p>
			</div>


			<div id="projects">
				<h2>Software projects</h2>
				
				<h4>
				shinyheatmap & fastheatmap
					<small>
						<a href="http://fastheatmap.com/">(Try it!)</a>
					</small>
				</h4>

				<p>
					Fastheatmap is a high performance interactive heatmap software system written in JavaScript and HTML.  It constitutes the HPC server behind <a href="http://shinyheatmap.com/">shinyheatmap</a>.  
					A good description of the project can be found here by: <a href="https://cloud.githubusercontent.com/assets/9893806/19628922/437136e8-9937-11e6-9516-2dac9d7652b0.png">Morpheus</a>.
					After you try it, you will feel like <a href="https://cloud.githubusercontent.com/assets/9893806/19628924/44e7168c-9937-11e6-9808-89452fbdd62d.png">Neo</a>.
				</p>
				
				<h4>
				SUPERmerge
					<small>
						<a href="https://github.com/Bohdan-Khomtchouk/SUPERmerge">(GitHub)</a>
					</small>
				</h4>

				<p>
					SUPERmerge is a ChIP-seq read pileup analysis and annotation tool written in the C programming language for investigating BAM files.  It lets you flexibly regulate a variety of read pileup parameters, thereby examining how read islands aggregate and what genomic regions they map to.  SUPERmerge is especially useful for investigating the characteristics of epigenetic histone modifications that have inherently broad peaks with a diffuse range of signal enrichment (e.g., H3K9me1, H3K27me3).
				</p>
				
				<h4>
				NGStoolkit
					<small>
						<a href="https://github.com/Bohdan-Khomtchouk/NGStoolkit">(GitHub)</a>
					</small>
				</h4>

				<p>
					NGStoolkit is a one-stop shop next-generation sequencing analysis toolkit written in Bash that allows you to run a complete NGS analysis pipeline (e.g., RNA-seq) by issuing just one command from the Terminal. Currently, NGStoolkit supports RNA-seq analysis via a single-click automated RNA-seq pipeline starting with bz2 files through differential expression.

					Current work is underway to expand NGStoolkit into ChIP-seq, methyl-seq, and exome-seq analysis territory to create a comprehensive NGS analysis toolkit.
				</p>
				
				<h4>
				Biochat
					<small>
						<a href="https://github.com/Bohdan-Khomtchouk/Biochat">(GitHub)</a>
					</small>
				</h4>
				<p>
					Biochat is a Python natural language processing (NLP) module written in highly-tuned Cython that efficiently categorizes and pairs similar items (e.g., words) into groups. It is the high performance computing (HPC) data science equivalent of the chemistry saying "like dissolves like."

					We apply the "like dissolves like" principle to teach data files to learn to talk to each other (quite literally). In order to talk, data must first be able to find each other in space (not a trivial task, considering that there are dozens of bioinformatics databases out there... see how we've tackled this problem with <a href="https://github.com/Bohdan-Khomtchouk/PubData">PubData</a>). So how, for example, is an RNA-seq dataset supposed to find its potentially related ChIP-seq dataset (e.g., according to some combination of similar cell type, histone mark, sequencing details, etc.)? Through metadata, of course! However, for the datasets to meet each other via a similar metadata footprint requires sophisticated NLP strategies to introduce them. Once the datasets meet, we can let the conversations (i.e., integrative bioinformatics analyses) begin!  Hence, we call this program Biochat.

					Our ultimate goal is to make integrative multi-omics a lot easier (and more fun) through artificial intelligence (AI). Right now, we are barely scratching the surface with NLP. Thus, we are currently implementing novel neural network approaches to help us teach data to talk to each other (stay tuned!).
				</p>

				<h4>
				Smalls
					<small>
						<a href="https://github.com/Bohdan-Khomtchouk/Smalls">(GitHub)</a>
					</small>
				</h4>
				<p>
					Smalls is a Python/R/shell scripting software program that efficiently aligns miRNA seed sequences to three-prime and five-prime untranslated regions in DNA (3'-UTR and 5'-UTR gene regions). Smalls' efficiency and flexibility stem from its utilization of advanced string algorithms and user-specified options for approximate matching (e.g., 2 mismatches).

					miRNA binding at UTR regions has been shown to influence biological processes as diverse as embryonic development or disease progression. It is, therefore, imperative to be able to computationally evaluate a candidate list of miRNAs in relation to their potential UTR binding targets. By evaluating which specific miRNAs align most often in the genome (as quantified by raw count of successful alignment at a given approximate mismatch rate), we can determine which specific miRNAs are most promising to experimentally validate in the lab. More importantly, we can determine the precise genomic position(s) of these miRNA-UTR alignments and, therefore, the identity of the respective gene(s) involved in the biological process at hand.

					Current work is underway to expand Smalls to cover other types of small RNAs in addition to miRNAs (e.g., snoRNAs, snRNAs, siRNAs, piRNAs, lncRNAs, vlincRNAs).
				</p>
				
				<h4>
				Triglav
					<small>
						<a href="https://github.com/Bohdan-Khomtchouk/Triglav">(GitHub)</a>
					</small>
				</h4>

				<p>
					Triglav is a bioinformatics software program written in the C programming language that performs the following operations:
					<ul>
						<li>
							Generates a single nucleotide polymorphism (SNP) list for every input file in your genomic sample pool
						</li>
						<li>
							Extracts variants based on each of these SNP lists
						</li>
						<li>
							Merges all extracted variants into one final file
						</li>
					</ul>
					Triglav outperforms the standard vcf-merge utility found in vcftools, which typically gives rise to problems when merging 
					indels with normal variant calling format (vcf) files. Also, the vcf-merge utility only takes the first values (in the INFO field) 
					from the list of merged vcf files, thereby affecting other post-analysis scripts such as VQSR for downstream filtering of the data.
					Current work is underway to transition Triglav from an HPC environment to a cloud infrastructure. Also, an indel-merge feature is 
					currently under development and testing.
				</p>
			</div>
			
			
			
			<div id="conferences">
				<h2>Conferences</h2>
				
				<h4>
				European Lisp Symposium, Brussels, Belgium ---  Spring 2017
				</h4>

				<p>
					Invited keynote oral presentation: "How the Strengths of Lisp-Family Languages Facilitate Building Complex and Flexible Bioinformatics Applications"
				</p>
				
				<h4>
				ISBRA ESBRA World Congress on Alcohol and Alcoholism, Berlin, Germany ---  Fall 2016
				</h4>

				<p>
					Poster presentation: "Dependence-induced increase of alcohol self-administration and compulsive drinking mediated by the histone methyltransferase PRDM2"
				</p>
				
				<h4>
				Miami Winter Symposium, Hyatt Regency Miami, Miami, FL ---  Winter 2016
				</h4>

				<p>
					Poster presentation: "Python and Apache Spark bioinformatics software program that sensitively detects the presence of snoRNAs and miRNAs within NGS samples"
				</p>
				
				<h4>
				Miami Winter Symposium, Hyatt Regency Miami, Miami, FL ---  Winter 2015
				</h4>

				<p>
					Poster presentation: "Personalized computational epigenomics pipeline to investigate complex brain tissue using ChIP-seq"
				</p>
				
				

			</div>	
			
			
			<div id="awards">
				<h2>Awards</h2>
				
				<h4>
				National Defense Science & Engineering Graduate Fellowship ---  2014-2017
				</h4>
				
				<h4>
				University of Miami Graduate Fellowship --- Fall 2013
				</h4>

				<h4>
				Who's Who Among Students in American Universities & Colleges Award ---  Spring 2016
				</h4>

			</div>	
			
			
			<div id="alumni">
				<h2>Undergraduate Alumni</h2>
				
				<h4>
				James R. Hennessy, B.Sc. ---  Current position: Data Analyst I, The Genome Technology Center, Stanford University School of Medicine
				</h4>
				<p>
					<ul> Years mentored: 2015-2017 </ul>
					<ul> Role: undergraduate student researcher (University of Miami, Department of Mathematics) </ul>
					<ul> Achievements: 2nd author of <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1260-x">MicroScope</a> </ul>
				</p>

			</div>		


			<div id="press">
				<h2>Press</h2>
				
				<h4>
				<a href="http://med.miami.edu/news/miller-school-scientists-publish-organic-chemistry-textbook">Miller School Scientists Publish Organic Chemistry Textbook</a>
				</h4>
				
				<h4>
				<a href="http://med.miami.edu/news/human-genetics-and-genomics-phd-student-awarded-prestigious-fellowship">Human Genetics and Genomics PhD Student Awarded Prestigious Fellowship</a>
				</h4>
				
				<h4>
				<a href="http://med.miami.edu/news/research-team-receives-nih-grant-for-study-of-epigenetic-processes-in-alcoh">Research Team Receives NIH Grant for Study of Epigenetic Processes in Alcoholism</a>
				</h4>

				<h4>
				<a href="http://www.huffingtonpost.co.uk/entry/alcohol- dependency- linked- to- lack- of- enzyme- prdm2_uk_57c55697e4b094071b4c947e">Alcoholics Lack Key Enzyme That Helps Control Impulses, Study Finds</a>
				</h4>
				
				<h4>
				<a href="http://www.wired.co.uk/article/alcoholism-rats-dependent-enzyme">Alcoholics lack a key enzyme in the brain - and this creates a 'vicious cycle' of addiction</a>
				</h4>
				
				<h4>
				<a href="https://www.sciencedaily.com/releases/2016/08/160830083707.htm">People with alcohol dependency lack important enzyme</a>
				</h4>
				
				<h4>
				<a href="https://www.thesun.co.uk/living/1695086/blame-your-brain-discovery-of-molecule-that-triggers-alcoholism-raises-hopes-of-a-cure/">ADDICTED TO BOOZE? Blame your brain! Discovery of molecule that 'triggers alcoholism' raises hopes of a cure</a>
				</h4>
				
				<h4>
				<a href="http://reliawire.com/addiction-enzyme/">PRDM2: Could Targeting Addiction Enzyme Help Fight Alcoholism?</a>
				</h4>

			</div>		
						
			
			
				<h2 id="contact">Contact</h2>
				<ul class="unstyled">
					<li>
						<i class="ai ai-google-scholar-square ai-3x"></i><strong> Google Scholar: </strong> <a href="https://scholar.google.com/citations?user=pa6Zh9QAAAAJ&hl=en">Bohdan Khomtchouk Google Scholar profile</a>
					</li>
					<li>
						<i class="icon-github"></i><strong> GitHub: </strong> <a href="https://github.com/Bohdan-Khomtchouk">github.com/Bohdan-Khomtchouk</a>
					</li>
					<li>
						<i class="icon-linkedin-sign"></i><strong> LinkedIn: </strong> <a href="https://www.linkedin.com/in/bohdankhomtchouk">linkedin.com/in/bohdankhomtchouk</a>
					</li>
					<li>
						<i class="icon-envelope"></i><strong> Email: </strong> khomtchoukmed@gmail.com
					</li>
				</ul>
			</div>

		</div>




<script type="text/javascript" src="js/jquery-1.9.1.min.js"></script>
<script type="text/javascript" src="bootstrap/js/bootstrap.min.js"></script>
</body>
</html>
